central/globalindex/mapping/mapping.go

```diff
@@ -146,24 +160,27 @@ func GetEntityOptionsMap() map[v1.SearchCategory]search.OptionsMap {
 		v1.SearchCategory_COMPLIANCE_CONTROL:    index.ControlOptions,
 		v1.SearchCategory_CLUSTERS:              clusterMapping.OptionsMap,
 		v1.SearchCategory_NAMESPACES:            namespaceMapping.OptionsMap,
-		v1.SearchCategory_NODES:                 nodeSearchOptions,
+		v1.SearchCategory_NODES:                 fullNodeOptionsMap,
 		v1.SearchCategory_PROCESS_BASELINES:     processBaselineMapping.OptionsMap,
 		v1.SearchCategory_REPORT_CONFIGURATIONS: reportConfigurationsMapping.OptionsMap,
 		v1.SearchCategory_RISKS:                 riskMappings.OptionsMap,
 		v1.SearchCategory_ROLES:                 roleOptions.OptionsMap,
 		v1.SearchCategory_ROLEBINDINGS:          roleBindingOptions.OptionsMap,
 		v1.SearchCategory_SERVICE_ACCOUNTS:      serviceAccountOptions.OptionsMap,
 		v1.SearchCategory_SUBJECTS:              subjectMapping.OptionsMap,
-		v1.SearchCategory_VULNERABILITIES:       cveSearchOptions,
-		v1.SearchCategory_COMPONENT_VULN_EDGE:   componentVulnEdgeMapping.OptionsMap,
-		v1.SearchCategory_CLUSTER_VULN_EDGE:     clusterVulnEdgeMapping.OptionsMap,
-		v1.SearchCategory_IMAGE_COMPONENT_EDGE:  imageComponentEdgeMapping.OptionsMap,
-		v1.SearchCategory_IMAGE_COMPONENTS:      componentSearchOptions,
-		v1.SearchCategory_IMAGE_VULN_EDGE:       imageCVEEdgeMapping.OptionsMap,
-		v1.SearchCategory_NODE_COMPONENT_EDGE:   nodeComponentEdgeMappings.OptionsMap,
```

Comment:
Edges should not be exposed.

Reply:
This is within GetEntityOptionsMap which gets called from here:
```
		for category, optMap := range GetEntityOptionsMap() {
			indexMapping.AddDocumentMapping(category.String(), blevesearch.DocumentMappingFromOptionsMap(optMap.Original()))
		}
```
Does this impact the persisted index in dackbox?

Reply:
not relevant anymore